src/Link.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Bootstrap5;
+
+final class Link
+{
+    private array $attributes = [];
+
+    public function __construct(
+        public readonly string $label = '',
+        public readonly string|null $url = null,
+    ) {
+    }
+
+    public function addAttributes(array $values): self
```

Comment:
```suggestion
    public function withAttributes(array $values): self
```

Reply:
This changes the essence and purpose of the method, which is clearly described, which is to add attributes, and we do not use with in widgets.

Reply:
Prefix `with` is incorrect, I was wrong. Need "attributes" name, because method replace attributes, not add.

Reply:
done.